logfire/_internal/integrations/llm_providers/openai.py

```diff
@@ -125,12 +125,7 @@ def get_response_data(self) -> Any:
 
     class OpenaiChatCompletionStreamState(StreamState):
         def __init__(self):
-            self._stream_state = ChatCompletionStreamState(
-                # We do not need the response to be parsed into Python objects so can skip
-                # providing the `response_format` and `input_tools` arguments.
-                input_tools=openai.NOT_GIVEN,
-                response_format=openai.NOT_GIVEN,
```

Comment:
This gave type errors in the newest version, the default changed, but I don't know why it didn't just lean on defaults before anyway.